elixir/README.md

```diff
@@ -0,0 +1,6 @@
+### Example of subscribing to the server and receiving a message, written in Elixir.
+
+#### Installation
+1. Erlang/OTP and Elixir
+2. Get dependencies and compile mix deps.get && mix compile
+3. Start mix run --no-halt
```

Comment:
You can improve readability by writing that as
```
2. Get dependencies and compile `mix deps.get && mix compile`
3. Start `mix run --no-halt`
```

Reply:
done